firebase-config/ktx/src/main/kotlin/com/google/firebase/remoteconfig/ktx/RemoteConfig.kt

```diff
@@ -45,6 +53,29 @@ fun remoteConfigSettings(
   return builder.build()
 }
 
+/**
+ * Starts listening for config updates from the Remote Config backend and emits [ConfigUpdate]s via
+ * a [Flow]. See [FirebaseRemoteConfig.addOnConfigUpdateListener] for more information.
+ *
+ * - When the returned flow starts being collected, an [ConfigUpdateListener] will be attached.
+ * - When the flow completes, the listener will be removed. If there are no attached listeners,
+ *   the connection to the Remote Config backend will be closed.
+ */
+fun FirebaseRemoteConfig.configUpdates(): Flow<ConfigUpdate> {
+  return callbackFlow {
```

Comment:
Since `configUpdates` doesn't have any arguments, it could probably be a property instead of a function:

```suggestion
val FirebaseRemoteConfig.configUpdates
  get() = callbackFlow<ConfigUpdate> {
```

Reply:
Good call